app/components/preprint-status-banner/template.hbs

```diff
@@ -22,8 +22,13 @@
         </div>
         <div class="dropdown reviewer-feedback">
             {{#bs-dropdown closeOnMenuClick=false as |dd|}}
-                {{#dd.button type="success"}}
-                    {{t labelDecisionDropdown}} <i class="fa fa-caret-down" aria-hidden="true"></i>
+                {{#dd.button type="success" disabled=submission.actions.isPending}}
+                    {{#if submission.actions.isPending}}
+                        <i class='fa fa-circle-o-notch fa-spin'></i>
```

Comment:
Can we add a little padding to the right of the icon?

Reply:
@laurenbarker done